game/game.go

```diff
@@ -522,6 +631,17 @@ func createGame(w ResponseWriter, r Request) (*Game, error) {
 	}
 	game.CreatedAt = time.Now()
 
+	if !game.NoMerge {
+		mergedWith, err := merge(ctx, r, game, user)
+		if err != nil {
+			return nil, err
+		}
+		if mergedWith != nil {
+			w.WriteHeader(http.StatusTeapot)
```

Comment:
I'm not sure what the right code should be here. Presumably it's a success, but it's probably not the expected success. Teapot sits fine with me.

Reply:
Haha great :D

Reply:
I started with 205 (StatusReset iirc), but a reply body wasn't allowed so I tried teapot instead. Which also didn't show a body!

Should I go back to 205?

Reply:
Teapot seems fine. If we get to a point where there are crowds of client devs objecting to the semantics of the API then we can revisit the decision, but both of us understand what it means so I think it's fine.

Reply:
Check.